03/styles/desktop.css

```diff
@@ -0,0 +1,8 @@
+body {
+    min-width: 1200px;
+    max-width: 1400px;
```

Comment:
Górną granicę zrozumiem, ale dolna to raczej nie najlepszy pomysł. 
Pamiętaj, że treść powinna się rozlewać - jak to w RWD